templates/addons/aurora/mysql-cf.yml

```
@@ -0,0 +1,171 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+  # Customize your Aurora Serverless cluster by setting the default value of the following parameters.
+  InitialDBName:
+    Type: String
+    Description: The name of the table.
```

Can we change this description to explain why we call it "InitialDBName?" 
```suggestion
    Description: The name of the table to be created in the DB cluster. 
```